### PR TITLE
new lesson POSTGRESQL implementing table relations

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

**Review by kcleland0818 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,32 +1,344 @@
-# Implementing Relations In SQL
+# Implementing Table Relations in SQL
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Implement a one-to-many relationship using an FOREIGN KEY constraint.
```
Comment (developerjaym) [inline README.md]: *an* should be *a*

File: README.md [hunk 2]
```diff
@@ -1,32 +1,344 @@
...
+  salary INTEGER,
+  department_id INTEGER NOT NULL,
+  CONSTRAINT department_fk FOREIGN KEY (department_id) REFERENCES department (id)
+);
+```
+
+The foreign key forces the database to check that each value in
+the `department_id` column corresponds to a value in the `id` column
+of the `department` table.  Thus, the following statement would
+result in a foreign key constrain violation as the `department` table
```
Comment (kcleland0818) [inline README.md]: Typo: "constrain" should be "constraint"

**Review by kcleland0818 — APPROVED** [review]
Small typo - everything else looks great!

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed

**Review by developerjaym — APPROVED** [review]

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**